node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/maintenance/coredump/CoredumpHandler.java

```diff
@@ -194,4 +204,47 @@ Path findCoredumpFileInProcessingDirectory(Path coredumpProccessingDirectory) {
                 .orElseThrow(() -> new IllegalStateException(
                         "No coredump file found in processing directory " + coredumpProccessingDirectory));
     }
+
+    void updateMetrics(NodeAgentContext context, Path containerCrashPathOnHost) {
+        Dimensions dimensions = generateDimensions(context);
+
+        // Unprocessed coredumps
+        int numberOfUnprocessedCoredumps = FileFinder.files(containerCrashPathOnHost)
```

Comment:
The `processing` directory may contain multiple files, at least the core file and `metadata.json`. You probably want to limit max-depth here, but then you'll be off by 1 if there is a coredump being processed right now...

Reply:
This is for number of coredumps in crash path, not the `processing` path. So I'll need to filter out hs_err files as well

Reply:
`processing` is inside the crash path
https://github.com/vespa-engine/vespa/blob/8bebe88526e449bbca0514d8bdfc87390cb26db4/node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/maintenance/coredump/CoredumpHandler.java#L87

Reply:
Aha 👍